tests/sdk/test_status.py

```diff
@@ -25,7 +25,7 @@ def callback():
         ext = Extension()
         ext.logger = MagicMock()
         ext._running_in_sim = True
-        ext._client = DebugClient("", "", MagicMock())
+        ext._client = DebugClient("", "", "", MagicMock())
```

Comment:
When secrets is optional this should not change

Reply:
Fixed in latest commit